packages/components/src/molecules/SkuSelector/SkuSelector.tsx

```diff
@@ -0,0 +1,135 @@
+import type { ChangeEventHandler, HTMLAttributes } from 'react'
+import React, { forwardRef, ReactNode } from 'react'
+import { Label } from '../..'
+
+interface SkuProps {
+  /**
+   * Alternative text description of the image. This is used on `renderImage` prop and it's mandatory if you want to use the `Image` variant.
+   */
+  alt?: string
+  /**
+   * Image URL. This is used on `renderImage` prop and it's mandatory if you want to use the `Image` variant.
+   */
+  src?: string
+  /**
+   * Current value for this option.
+   */
+  value?: string
+  /**
+   * URL for option's page. This is used on `renderLink` prop.
+   */
+  href: string
+  /**
+   * Hex color code for this option. This is mandatory if you want to use the `Color` variant.
+   */
+  hexColor?: string
+  /**
+   * Specifies that this option should be disabled.
+   */
+  disabled?: boolean
+  /**
+   * A React component that will be rendered as a link.
+   */
+  renderLink: (href: string) => ReactNode
+  /**
+   * Function to define what image should be used.
+   */
+  renderImage?: (src?: string, alt?: string) => ReactNode
+}
+
+// TODO: Add the 'color' variant back once the store supports naturally handling color SKUs.
+type Variant = 'label' | 'image' | 'color'
+
+export interface SkuSelectorProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * Specify which variant the component should handle.
+   */
+  variant: Variant
+  /**
+   * SKU options that should be rendered.
+   */
+  options: SkuProps[]
+  /**
+   * Section label for the SKU selector.
+   */
+  label?: string
+  /**
+   * Currently active variation's value.
+   */
+  activeValue: string
+  /**
+   * Function to be triggered when SKU option change.
+   */
+  onChange?: ChangeEventHandler<HTMLInputElement>
```

Comment:
I think we can remove this function 🤔 . Since now we're not using elements that would naturally receive onChange events, this function will never be called.